apigw-lambda-sfn-transcribe-translate-polly-sam/src/invokeTranslator.js

```diff
@@ -25,12 +26,11 @@ module.exports.handler = async (event, context) => {
       stateMachineArn: process.env.statemachine_arn,
       input: body
   }
-  let response = await stepfunctions.startSyncExecution(params).promise();
-  await apig
-    .postToConnection({
+  let response = await stepfunctions.send(new StartSyncExecutionCommand(params));
+  await apig.send(new PostToConnectionCommand({
```

Comment:
note: Some error occurs. Because in Node.js 18 and later runtimes, AWS SDK v3 is bundled, so I rewrote it to v3. See articles below.

- https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/
- https://aws.amazon.com/blogs/compute/node-js-22-runtime-now-available-in-aws-lambda/